napari/_qt/layers/qt_image_layer.py

```diff
@@ -114,118 +124,52 @@ def __init__(self, layer):
         self.grid_layout.setColumnStretch(1, 1)
         self.grid_layout.setSpacing(4)
 
-    def changeInterpolation(self, text):
+    def _on_interpolation_change(self, text):
         """Change interpolation mode for image display.
 
-        Parameters
-        ----------
-        text : str
-            Interpolation mode used by vispy. Must be one of our supported
-            modes:
-            'bessel', 'bicubic', 'bilinear', 'blackman', 'catrom', 'gaussian',
-            'hamming', 'hanning', 'hermite', 'kaiser', 'lanczos', 'mitchell',
-            'nearest', 'spline16', 'spline36'
-        """
-        self.layer.interpolation = text
-
-    def changeRendering(self, text):
-        """Change rendering mode for image display.
-
-        Parameters
-        ----------
-        text : str
-            Rendering mode used by vispy.
-            Selects a preset rendering mode in vispy that determines how
-            volume is displayed:
-            * translucent: voxel colors are blended along the view ray until
-              the result is opaque.
-            * mip: maxiumum intensity projection. Cast a ray and display the
-              maximum value that was encountered.
-            * additive: voxel colors are added along the view ray until
-              the result is saturated.
-            * iso: isosurface. Cast a ray until a certain threshold is
-              encountered. At that location, lighning calculations are
-              performed to give the visual appearance of a surface.
-            * attenuated_mip: attenuated maxiumum intensity projection. Cast a
-              ray and attenuate values based on integral of encountered values,
-              display the maximum value that was encountered after attenuation.
-              This will make nearer objects appear more prominent.
-        """
-        self.layer.rendering = text
-        self._toggle_rendering_parameter_visbility()
-
-    def changeIsoThreshold(self, value):
-        """Change isosurface threshold on the layer model.
-
-        Parameters
-        ----------
-        value : float
-            Threshold for isosurface.
-        """
-        with self.layer.events.blocker(self._on_iso_threshold_change):
-            self.layer.iso_threshold = value / 100
+       Parameters
+       ----------
+       text : str
+           Interpolation mode used by vispy. Must be one of our supported
+           modes:
+           'bessel', 'bicubic', 'bilinear', 'blackman', 'catrom', 'gaussian',
+           'hamming', 'hanning', 'hermite', 'kaiser', 'lanczos', 'mitchell',
+           'nearest', 'spline16', 'spline36'
```

Comment:
Side note: how do we feel about trimming this list way down? I feel like there's really no need for us to provide all these, and the list is confusing. Alternatively, we could sort into two lists with a [separator](https://doc.qt.io/qtforpython/PySide2/QtWidgets/QComboBox.html#PySide2.QtWidgets.PySide2.QtWidgets.QComboBox.insertSeparator). First "nearest", "bilinear", "bicubic", and the second containing the rest?